macros/graph/plotly3D.pl

```diff
@@ -265,26 +262,25 @@ sub HTML {
 	my $title = ($self->{title}) ? "<strong>$self->{title}</strong>" : '';
 	my $plots = '';
 	my @data  = ();
-	my $count = 0;
 	my $scene = ($self->{scene}) ? "scene: { $self->{scene} }," : '';
 
 	foreach (@{ $self->{plots} }) {
-		$count++;
-		$plots .= $_->HTML($id, $count);
-		push(@data, "plotlyData${id}_$count");
+		my $thisId = $_->{id};
+		$plots .= $_->HTML;
+		push(@data, "plotlyData_$thisId");
```

Comment:
There is no need for `$thisId`.  Just change this to:
```suggestion
		$plots .= $_->HTML;
		push(@data, "plotlyData_$_->{id}");
```

Reply:
fixed.